census/loaders/index.js

```diff
@@ -210,16 +216,12 @@ var loadData = function(options, models) {
     .spread((data, site) => {
       return Promise.all(_.map(data, dataObj => {
         // Allow custom data mapping
-        let createData = _.chain(
-          _.isFunction(options.mapper) ? options.mapper(dataObj, site) : dataObj
-        )
+        let createData = _.isFunction(options.mapper) ?
+          options.mapper(dataObj, site) : dataObj;
```

Comment:
Agreed - multi-lining in JS doesn't have solid conventions, so mine can be a bit hit and miss :)